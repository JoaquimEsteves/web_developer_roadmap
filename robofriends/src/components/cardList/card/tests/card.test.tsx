import React from 'react';


import { shallow } from 'enzyme';
import Card from '../card';
import '../../../../setupTests';

it('try out the snapshot yo', () => {
    const example = { id: 1, name: "", email: "", username: "" };
    expect(shallow(<Card {...example} />).debug()).toMatchSnapshot();
})