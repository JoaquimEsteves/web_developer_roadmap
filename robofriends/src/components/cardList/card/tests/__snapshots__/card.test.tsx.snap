// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`try out the snapshot yo 1`] = `
"<div className=\\"card\\">
  <img src=\\"https://robohash.org/1?size=200x200\\" alt=\\"card\\" />
  <div>
    <h2 />
    <p />
  </div>
</div>"
`;
